diagnostic/server.go

```diff
@@ -103,19 +103,19 @@ func (s *Server) EnableDebug(ip string, port int) {
 	}(s)
 }
 
-// DisableDebug stop the dubug and closes the tcp socket
-func (s *Server) DisableDebug() {
+// DisableDiagnostic stop the dubug and closes the tcp socket
+func (s *Server) DisableDiagnostic() {
 	s.Lock()
 	defer s.Unlock()
 
 	s.srv.Shutdown(context.Background())
 	s.srv = nil
 	s.enable = 0
-	logrus.Info("Disabling the diagnose server")
+	logrus.Info("Disabling the diagnostic server")
 }
 
-// IsDebugEnable returns true when the debug is enabled
-func (s *Server) IsDebugEnable() bool {
+// IsDiagnosticEnable returns true when the debug is enabled
+func (s *Server) IsDiagnosticEnable() bool {
```

Comment:
Minor nit: how about changing the name to `IsDiagnosticEnable**d**` like in controller
```func (c *controller) IsDiagnosticEnabled() bool {```

Reply:
sure